### PR TITLE
Catalog  , cart and wishlist

### PR DESCRIPTION
**Pull request by vashnavichauhan1825**

### CODE REVIEW

File: catelog.css [hunk 1]
```diff
@@ -0,0 +1,274 @@
+@import url('https://vaspacx-ecommerce.netlify.app/main.css');
+
+body{
+    height:auto;
+}
+
+aside{
+    width: 12vw;
+    /* background-color: var(--secbg-color); */
```
Comment (Dev-AkshatGupta) [inline catelog.css]: Hey remove the commented code and also add some comments on the code to enhance developer experience

File: catelog.css [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+.cart-label{
+    display: grid;
+    grid-template-columns: 506px 266px 1fr;
+}
+
+
+
+.cartitem-one{
+    border-bottom: 1px solid var(--lightGrey);
+    /* background-color: var(--secbg-color); */
```
Comment (Dev-AkshatGupta) [inline catelog.css]: Again please remove the commented code

File: catelog.css [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+/* cart summary */
+
+.cart-sum{
+    background-color: var(--secbg-color);
+    padding: 25px;
+    width: 284px;
+    border-radius: 10px;
+    height: 402px;
+    display: grid;
+    grid-template-rows: 86px 124px 50px 79px;
```
Comment (Dev-AkshatGupta) [inline catelog.css]: Suggestion - Try to avoid the Hard values

File: catelog.css [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+    }
+
+
+.cart-sum div button{
+    margin-left: 78px;
```
Comment (Dev-AkshatGupta) [inline catelog.css]: I guess you would have used something flex-end . Please try if you can do so

File: catelog.css (hunk 1 above)
Comment (vashnavichauhan1825) [reply catelog.css]: yes sure

File: catelog.css (hunk 2 above)
Comment (vashnavichauhan1825) [reply catelog.css]: i will rectify it

File: catelog.css [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+    margin-top: 13px;
+}
+
+.button-shopping{
+    background: transparent;
+    color: var(--font-primary);
+    border: 2px solid var(--lightGrey);
+    border-radius: 5px;
+    padding: 10px;
+    margin-top: 66px;
```
Comment (Dev-AkshatGupta) [inline catelog.css]: same here try to avoid hard coded values

File: catelog.css [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+    align-items: center;
+    padding:15px;
+}
+
+.item-name ul{
+list-style:none;
+}
+
+
+
```
Comment (b0llu) [inline catelog.css]: remove the extra spacing

File: catelog.css [hunk 7]
```diff
@@ -0,0 +1,274 @@
...
+.wishlist-item-container .product-display{
+    width: auto;
+    height: auto;
+    position: relative;
+    cursor: pointer;
+    }
+    
+   
+
```
Comment (b0llu) [inline catelog.css]: remove the extra spacing

**Pull request merged**